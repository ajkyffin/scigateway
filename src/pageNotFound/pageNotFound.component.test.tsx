import React from 'react';
import PageNotFound from './pageNotFound.component';
import { createShallow } from '@material-ui/core/test-utils';

describe('Page Not found component', () => {
  let shallow;

  beforeEach(() => {
    shallow = createShallow({});
  });

  it('pageNotFound renders correctly', () => {
    const wrapper = shallow(<PageNotFound />);
    expect(wrapper.dive()).toMatchSnapshot();
  });
});
