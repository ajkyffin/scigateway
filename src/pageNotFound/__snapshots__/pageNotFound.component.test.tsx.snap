// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Not found component pageNotFound renders correctly 1`] = `
<div>
  <div
    className="PageNotFound-titleContainer-1"
  >
    <BugReportIcon
      className="PageNotFound-bugIcon-2"
    />
    <WithStyles(ForwardRef(Typography))
      className="PageNotFound-codeText-3"
    >
      404
    </WithStyles(ForwardRef(Typography))>
  </div>
  <div
    className="PageNotFound-container-4"
  >
    <WithStyles(ForwardRef(Typography))
      variant="h2"
    >
      Page not found
    </WithStyles(ForwardRef(Typography))>
    <WithStyles(ForwardRef(Typography))
      className="PageNotFound-message-5"
      variant="body1"
    >
      We're sorry, the page you requested could not be found. Please check the page address for typos. Click 
      <Link
        to="/"
      >
        here
      </Link>
       to return to the home page or contact support.
    </WithStyles(ForwardRef(Typography))>
  </div>
</div>
`;
